defaults/main.yml

```diff
@@ -19,26 +18,49 @@ libvirt_vm_engine:
 # correct emulator to use.
 libvirt_vm_emulator:
 
-# CPU architecture.
-libvirt_vm_arch: x86_64
+# A list of specifications of VMs to be created.
+# For backwards compatibility, libvirt_vms defaults to a singleton list using
+# the values of the deprecated variables below.
+# See README.md or tasks/main.yml for these attributes' defaults.
+libvirt_vms:
+      # State of the VM. May be 'present' or 'absent'.
+    - state: "{{ libvirt_vm_state }}"
```

Comment:
I still don't think the defaults will work correctly in this case, since the variable is defined - it will be None. How about just keeping the old defaults on the deprecated variables?

Reply:
Yeah, true. It seems more consistent to use the boolean flag in the default filter for all the variables though, rather than putting some of the defaults in here.